src/esp/bindings/GeoBindings.cpp

```diff
@@ -64,6 +66,112 @@ void initGeoBindings(py::module& m) {
            1.0 is a chordal Catmull-Rom spline)",
       "key_points"_a, "num_interpolations"_a, "alpha"_a = .5f);
 
+  // ==== VoxelWrapper ====
```

Comment:
Could all these new bindings be a separate PR? This PR with core VoxelGrid implementation is already very large.